test/common.js

```diff
@@ -57,6 +57,20 @@ exports.validate = (t, report, options) => {
                 new RegExp('Node.js version: ' + process.version),
                 'NodeReport section contains expected Node.js version');
       }
+      if (options && options.commandline) {
+        if (this.isWindows()) {
+          // On Windows we need to strip double quotes from the command line in
+          // the report, and escape backslashes in the regex comparison string.
```

Comment:
Ah I think it might be spaces in the paths -- I get the same if I use Node.js installed under `C:\Program Files`